packages/core/src/external/quest/api/patient-mapping.ts

```diff
@@ -20,7 +20,7 @@ export async function getPatientMapping(
 ): Promise<PatientMapping> {
   const { log } = out(`Quest getPatientMapping - externalId ${externalId}`);
   const api = axiosInstance ?? axios.create({ baseURL: Config.getApiUrl() });
-  const getPatientMappingUrl = `/internal/quest/patient/${externalId}`;
+  const getPatientMappingUrl = `/internal/quest/patient-mapping/${externalId}`;
```

Comment:
Where is this endpoint being used/called/stored?

re: endpoint path

nit: I'd keep it `/internal/quest/patient/mapping?externalId=<external-id>`